lib/internal/crypto/keys.js

```diff
@@ -509,12 +512,67 @@ function getKeyTypes(allowKeyObject, bufferOnly = false) {
   return types;
 }
 
+const oids = {
+  'ML-DSA-44': [96, 134, 72, 1, 101, 3, 4, 3, 17],
+  'ML-DSA-65': [96, 134, 72, 1, 101, 3, 4, 3, 18],
+  'ML-DSA-87': [96, 134, 72, 1, 101, 3, 4, 3, 19],
+};
+
+function mlDsaPubLen(alg) {
+  switch (alg) {
+    case 'ML-DSA-44': return 1312;
+    case 'ML-DSA-65': return 1952;
+    case 'ML-DSA-87': return 2592;
+  }
+}
+
 function getKeyObjectHandleFromJwk(key, ctx) {
   validateObject(key, 'key');
-  validateOneOf(
-    key.kty, 'key.kty', ['RSA', 'EC', 'OKP']);
+  if (EVP_PKEY_ML_DSA_44 || EVP_PKEY_ML_DSA_65 || EVP_PKEY_ML_DSA_87) {
+    validateOneOf(
+      key.kty, 'key.kty', ['RSA', 'EC', 'OKP', 'AKP']);
+  } else {
+    validateOneOf(
+      key.kty, 'key.kty', ['RSA', 'EC', 'OKP']);
+  }
   const isPublic = ctx === kConsumePublic || ctx === kCreatePublic;
 
+  if (key.kty === 'AKP') {
+    validateOneOf(
+      key.alg, 'key.alg', ['ML-DSA-44', 'ML-DSA-65', 'ML-DSA-87']);
+    validateString(key.pub, 'key.pub');
+
+    if (isPublic) {
+      const keyData = Buffer.from(key.pub, 'base64url');
+      if (keyData.byteLength !== mlDsaPubLen(key.alg)) {
+        throw new ERR_CRYPTO_INVALID_JWK();
+      }
+      const handle = new KeyObjectHandle();
+
+      if (!handle.initMlDsaPublicRaw(key.alg, keyData)) {
+        throw new ERR_CRYPTO_INVALID_JWK();
+      }
+
+      return handle;
+    }
+
+    validateString(key.priv, 'key.priv');
+    const keyData = Buffer.from(key.priv, 'base64url');
+    if (keyData.byteLength !== 32) {
+      throw new ERR_CRYPTO_INVALID_JWK();
+    }
+
+    const bytes = [48, 52, 2, 1, 0, 48, 11, 6, 9, ...oids[key.alg], 4, 34, 128, 32, ...keyData];
+
+    const handle = new KeyObjectHandle();
+    try {
+      handle.init(kKeyTypePrivate, new Uint8Array(bytes), kKeyFormatDER, kKeyEncodingPKCS8, null);
```

Comment:
OpenSSL doesn't seem to have a native function that fits our handle.init that would allow for just setting the raw `seed`, given these are fixed length keys without variable encodings lengths I just went the route of constructing a der wrapper and using that.